docs/developer_guide.md

```diff
@@ -148,11 +150,6 @@ steps:
 4. Select "All Platforms" in the Platform dropdown.
 5. Select "Configuration Properties", then "C/C++", then "Preprocessor" in the 
 left tree structure.
-6. In the entry called "Preprocessor Definitions":
-    * Add "DISABLE_TELEMETRY_TRACELOGGING;" to disable Microsoft telemetry 
-    logging alone. 
-    * Add "DISABLE_PERF_TRACELOGGING;" to disable performance logging alone.
-    * Add "DISABLE_DEBUG_TRACELOGGING;" to disable debug logging alone.
-    * Or simply add "DISABLE_ALL_TRACELOGGING;" to disable all three types of logging.
+6. In the entry called "Preprocessor Definitions", add "EMIT_TELEMETRY_EVENTS;".
 7. Click the "Apply" button.
```

Comment:
Is it possible to instead guide people to add the EmitTelemetryEvents environment variable? Just in case we even have more than just a preprocessor define control this?

Reply:
Updated the instructions such that they point to the EmitTelemetryEvents environment variable instead.

Reply:
Oh ... hmm. I guess that is one way of doing it. I really thought that VS had a dialog for setting environment variables for the build that it does, but I can't find it. This can do it and affect only the build you're doing:

1) Open command prompt and set the environment variable
2) Run "MUXControls.sln" from the command line to launch VS (VS will pick up the environment variables you set from the command prompt this way).

Reply:
I believe older versions of Visual Studio indeed offered a UI for setting environment variables. But the new version does not, as far as I can tell. That's why I ended up with that method.

But you're right that setting the variable in a Command Prompt and launching the solution from there works. I verified that.
Guidance is updated accordingly.